<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPView</class>
 <widget class="QMainWindow" name="BPView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>UTM Hanzel</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	border-image:url(:/img/UI/images/Custom Size - 6.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>UTM Hanzel</family>
    </font>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>600</y>
      <width>251</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSwap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
background-image: url(:/img/UI/images/SELEX SBS icon/Group 371.png);
background-repeat: no-repeat;
text-decoration: none;
background-position:left top;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
background-image: url(:/img/UI/images/SELEX SBS icon/Group 370.png);
background-repeat: no-repeat;
text-decoration: none;
background-position:left top;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>600</y>
      <width>391</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelSerialnumber">
       <property name="minimumSize">
        <size>
         <width>216</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>UTM BryantLG</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(141, 198, 255);</string>
       </property>
       <property name="text">
        <string>Số Seri</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblBatSN">
       <property name="minimumSize">
        <size>
         <width>159</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>UTM BryantLG</family>
         <pointsize>21</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>BP-125487</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTemp">
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>UTM BryantLG</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(141, 198, 255);</string>
         </property>
         <property name="text">
          <string>Nhiệt độ pin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lblSOH">
         <property name="minimumSize">
          <size>
           <width>159</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>UTM BryantLG</family>
           <pointsize>21</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(32, 74, 135)</string>
         </property>
         <property name="text">
          <string>90%</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblTemp">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>159</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>UTM BryantLG</family>
           <pointsize>21</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>BlankCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(32, 74, 135)</string>
         </property>
         <property name="text">
          <string>30°C</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelSOH">
         <property name="font">
          <font>
           <family>UTM BryantLG</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(141, 198, 255);</string>
         </property>
         <property name="text">
          <string>Độ bền pin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelT">
       <property name="minimumSize">
        <size>
         <width>216</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>UTM BryantLG</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(141, 198, 255);</string>
       </property>
       <property name="text">
        <string>Chu kỳ nạp xả còn lại</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCycles">
       <property name="minimumSize">
        <size>
         <width>159</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>UTM BryantLG</family>
         <pointsize>21</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135)</string>
       </property>
       <property name="text">
        <string>200</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelSOC">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>233</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>159</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>1677770</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>UTM BryantLG</family>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(211, 215, 207);
background: transparent;
background-repeat: no-repeat;
text-decoration: none;</string>
    </property>
    <property name="text">
     <string>20%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelCabinNum">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>258</y>
      <width>114</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>UTM BryantLG</family>
      <pointsize>67</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 224);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="timeoutBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1250</y>
      <width>800</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk {
	background-color: 	qlineargradient( x1: 0, y1 : 0, x2: 1,  y2: 0, stop:0.8 cyan, stop:0.9 white, stop: 0.95 cyan );		
	border-radius:4px;
}
QProgressBar {
	border: 0px; 
	border-radius: 4px;
	background: rgb(136, 138, 133); 
	padding: 0px; 
	margin-right: 4px;
}</string>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="value">
     <number>20000</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPicPower">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>420</y>
      <width>237</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/UI/images/SELEX SBS icon/Group 408.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCountDowTimeout">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>1217</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>UTM BryantLG</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(15, 45, 208);</string>
    </property>
    <property name="text">
     <string>Thời gian còn lại:20giây</string>
    </property>
   </widget>
   <zorder>labelPicPower</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>labelSOC</zorder>
   <zorder>labelCabinNum</zorder>
   <zorder>timeoutBar</zorder>
   <zorder>labelCountDowTimeout</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
